### PR TITLE
change the default ttl to infinite

### PR DESCRIPTION
**Pull request by YiMysty**
Things to consider before you submit the PR:

* [ ] Has `CHANGELOG.md` been updated?
* [x] Are tests passing locally?
* [x] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you check UI tests on the sample app? They are not executed on CI.
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

https://dev.azure.com/msmobilecenter/Mobile-Center/_workitems/edit/63664

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: AppCenterData/AppCenterData/MSData.h [hunk 1]
```diff
@@ -44,7 +44,7 @@ static int const kMSDataTimeToLiveNoCache = 0;
 /**
  * Default expiration on cache.
  */
-static int const kMSDataTimeToLiveDefault = 60 * 60 * 24;
+static int const kMSDataTimeToLiveDefault = kMSDataTimeToLiveInfinite;
```
Comment (MehrdadMzfr) [inline AppCenterData/AppCenterData/MSData.h]: both are consts, why don't you use the kMSDataTimeToLiveInfinite directly?

Comment (YiMysty) [reply AppCenterData/AppCenterData/MSData.h]: talked privately :) to avoid breaking change, we will keep it~

File: CHANGELOG.md [hunk 2]
```diff
@@ -15,14 +15,10 @@
 
 * **[Feature]** Now supports tvOS.
 
-___
-
-## Version 2.2.0 (Under development)
```
Comment (YiMysty) [inline CHANGELOG.md]: no idea why it contains two versions header, removed one...

File: CHANGELOG.md [hunk 3]
```diff
@@ -15,14 +15,10 @@
...
-
 ### AppCenterData
```
Comment (YiMysty) [inline CHANGELOG.md]: is apple sdk also space friendly? 🤔

File: AppCenterData/AppCenterData/MSBaseOptions.h [hunk 4]
```diff
@@ -6,7 +6,7 @@
 @interface MSBaseOptions : NSObject
 
 /**
- * Device document time-to-live in seconds. Default is one day.
+ * Device document time-to-live in seconds. Default is infinite.
```
Comment (dhei) [inline AppCenterData/AppCenterData/MSBaseOptions.h]: Let's not hard code the values in comment as they are easy to get out of date.

```suggestion
 * Device document time-to-live in seconds.
```

Comment (YiMysty) [reply AppCenterData/AppCenterData/MSBaseOptions.h]: oh oh~ sure!

File: CHANGELOG.md [hunk 5]
```diff
@@ -15,14 +15,10 @@
...
 
 * **[Feature]** Add support for offline list of documents.
-
+* **[Feature]** Change the default offline persistence from 1 day to infinite.
```
Comment (dhei) [inline CHANGELOG.md]: Let's use the same terminology `time-to-live (TTL)` as in the docs (https://docs.microsoft.com/en-us/appcenter/sdk/data/ios).

```suggestion
* **[Feature]** Change the default device time-to-live (TTL) from 1 day to infinite (never expire).
```

Comment (YiMysty) [reply CHANGELOG.md]: woo.. how about remove `device` in this sentence? :)

File: CHANGELOG.md (hunk 3 above)
Comment (dhei) [reply CHANGELOG.md]: This looks fine.

**Comment by YiMysty** [comment]
merged to unblock xamarin work~ feel free to left comment if found more :)

**Pull request merged**